src/content/en/tools/workbox/get-started/gulp.md

```diff
@@ -1,51 +1,188 @@
-project_path: /web/_project.yaml
+project_path: /web/tools/_project.yaml
 book_path: /web/tools/_book.yaml
-description: How to use Workbox with Gulp.
-
-{# wf_updated_on: 2017-10-14 #}
-{# wf_published_on: 2017-10-14 #}
-
-# Gulp {: .page-title }
-
-Use gulp and `workbox-build` to build a precaching service worker. Simply
-install the module then cut and paste the code sample.
-
-**1.** [Install Node.js](https://nodejs.org/en/).
-
-**2.** Install the module with NPM.
-
-```bash
-npm install workbox-build --save-dev
-```
-
-**3.** Require `workbox-build` in your gulp `gulpfile.js`.
-
-```javascript
-const wbBuild = require('workbox-build');
-```
-
-**4.** Also in `gulpfile.js` add a task to build a service worker.
-
-```javascript
-gulp.task('bundle-sw', () => {
-  return wbBuild.generateSW({
-    globDirectory: './app/',
-    swDest: './app/sw.js',
-    globPatterns: ['**\/*.{html,js,css}'],
-    globIgnores: ['admin.html'],
-    templatedUrls: {
-      '/shell': ['shell.hbs', 'main.css', 'shell.css'],
-    },
-  })
-  .then(() => {
-    console.log('Service worker generated.');
-  })
-  .catch((err) => {
-    console.log('[ERROR] This happened: ' + err);
-  });
-})
-```
-
-**Note:** The gulp task for generating the service worker should always be
-run as the last step in each build. This ensures that your service worker
-contains any changes made during development.
+description: Learn how to make a gulp-based app work offline by adding Workbox to it.
+
+{# wf_updated_on: 2017-11-17 #}
+{# wf_published_on: 2017-11-17 #}
+
+# Get Started With Workbox For Gulp {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+In this tutorial, you use Workbox to make a simple web app work offline.
+
+If you'd like a conceptual overview of Workbox before starting this tutorial,
+see the [Overview](/web/tools/workbox/overview).
+
+## Step 1: Set up your project {: #setup }
+
+The project that you're going to add Workbox to is hosted on [Glitch][Glitch].
+First, you need to set up Glitch so that you can edit your own copy of the
+project.
+
+[Glitch]: https://glitch.com/about/
+
+1. Open the [demo](https://glitch.com/edit/#!/workbox-gulp).
```

Comment:
FYI, there's a weird `.v8flags` file in the Glitch project right now that you should delete.